database/sql/V002__create_trades.sql

```diff
@@ -0,0 +1,14 @@
+-- Trade events from the smart contract.
```

Comment:
I guess you split it into files just to try out the migrations

I think is fine, but also there was nothing wrong with having all the initial database in one file, and u add other files when u need to do the actual migrations.